tsdb/enginetags.go.go

```diff
@@ -0,0 +1,29 @@
+package tsdb
```

Comment:
Is the `.go.go` extension here intentional?

Reply:
Thanks! fixed